test/usecases/get_auth_token_use_case_test.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter_survey/database/secure_storage.dart';
+import 'package:flutter_survey/usecases/get_auth_token_use_case.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import '../mocks/generate_mocks.mocks.dart';
+
+void main() {
+  group('GetAuthTokenUseCaseTest', () {
+    late MockSecureStorage mockSecureStorage;
+    late GetAuthTokenUseCase getAuthTokenUseCase;
+
+    setUp(() async {
+      mockSecureStorage = MockSecureStorage();
+      getAuthTokenUseCase = GetAuthTokenUseCase(mockSecureStorage);
+    });
+
+    test('When fetch auth token data it returns accordingly', () async {
```

Comment:
Let's add the comma for consistency
```suggestion
    test('When fetch auth token data, it returns accordingly', () async {
```